### DISCUSSION TITLE
[IDEA]

### DISCUSSION THREAD

**Discussion by Sandhyateju** [khoj-ai/khoj#767] [Q&A]
## Describe the feature you'd like
A clear and concise description of what you want to happen. Include any relevant links or screenshots or inspiration.

**Comment by sabaimran** [comment]
Hi @Sandhyateju ! Could you include a description of what you're looking for?

**Reply by Adnanaboobacker** [reply]
Real life applications,scenarios connecting other engineering disciplines,governance and solving social issues

**Comment by Sandhyateju** [comment]
How can I use this khoj-ai ?

**Comment by debanjum** [comment]
@Sandhyateju can you provide more details on what you mean? To quickly get started using Khoj you can just signup on [app.khoj.dev](https://app.khoj.dev/login?) with a Google account. Please refer to the [documentation](https://docs.khoj.dev) for more details on how to use Khoj.

P.S: Converting this into a discussion for now
